internal/jimm/sshkeys/sshkeys_test.go

```diff
@@ -93,6 +93,73 @@ func (s *sshKeysManagerSuite) TestListUserPublicKeys(c *qt.C) {
 	c.Assert(keys[0].Marshal(), qt.DeepEquals, s.pubKey.Marshal())
 }
 
+func (s *sshKeysManagerSuite) TestVerifyPublicKeys(c *qt.C) {
+	c.Parallel()
+	ctx := context.Background()
+
+	err := s.manager.AddUserPublicKey(ctx, s.user, s.pubKey)
+	c.Assert(err, qt.IsNil)
+
+	key, err := rsa.GenerateKey(rand.Reader, 2048)
```

Comment:
The variable names are tricky to read, maybe `rawKey` for the private keys (or just privKey)